addons/website_sale/controllers/main.py

```diff
@@ -423,6 +423,8 @@ def shop(self, page=0, category=None, search='', min_price=0.0, max_price=0.0, p
             'search': fuzzy_search_term or search,
             'original_search': fuzzy_search_term and search,
             'order': post.get('order', ''),
+            'start_date': post.get('start_date', ''),
+            'end_date': post.get('end_date', ''),
```

Comment:
Hello @rupa-odoo 

It's the same for those variables. You need to overrides `shop` in `website_sale_renting`.

Reply:
Yes, I had made that change already but I forgot to push that change sorry :cry:

Reply:
Done the changes :+1: 
Enterprise PR : https://github.com/odoo/enterprise/pull/64962